index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+    <h1>Hello Microverse!</h1>
+    </header>
```

Comment:
- [ ] You're required to give a class name to your `h1` element and use the class name to style the content of the `h1` tag.

E.g
```
  index.html
   <h1 class="title">Hello Microverse!</h1>

 style.css
  .title {
     color: blue;
   }
```